docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -21,6 +21,9 @@ peer recoveries:
     the cluster. Defaults to `40mb`.
 
 `indices.recovery.max_concurrent_file_chunks`::
+    Controls the number of file chunk requests can be sent in parallel per recovery.
+    Since the default value is good enough to saturate a network pipe efficiently;
```

Comment:
Perhaps change this to: "As multiple recoveries are already running in parallel (controlled by `cluster.routing.allocation.node_concurrent_recoveries`), increasing this expert-level setting might only help in situations where peer recovery of a single shard is not reaching the total inbound and outbound peer recovery traffic as configured by `indices.recovery.max_bytes_per_sec`, but is CPU-bound instead, typically when using transport-level security or compression."